joern-cli/src/test/scala/io/joern/joerncli/JoernExportTests.scala

```diff
@@ -15,7 +15,14 @@ class JoernExportTests extends AnyWordSpec with Matchers with AbstractJoernCliTe
       val tempDir = os.temp.dir(prefix = "joern-export-test")
       JoernExport.exportCpg(cpg, Representation.Cpg, JoernExport.Format.Graphson, tempDir.toNIO)
       val exportedFiles = os.walk(tempDir).filter(_.toIO.isFile)
-      exportedFiles.size should be >= 100
+
+      // c frontend creates a different cpg on windows - not sure about the details, but purely for the export it doesn't
```

Comment:
The windows GH action runner does not provide a gcc installation. Thus, scanning system header files results in 0 files. You may fix this by running the c2cpg with `--no-include-auto-discovery`. Maybe we should make this the default as it pops up every now and then (leads to different CPGs on different environments all the time).

Reply:
@mpollmeier There is https://github.com/joernio/joern/pull/1909 for this.

Reply:
👍🏻 I see
re https://github.com/joernio/joern/pull/1909 i'll drop the distinction in this test then